transformations/stopword_removal/README.md

```diff
@@ -0,0 +1,15 @@
+# Stopword Removal
+Removes stopwords from a piece of text.
+
+Author: Juan Yi Loke
+Email: juanyi.loke@mail.utoronto.ca
+Affliation: University of Toronto
+
```

Comment:
I believe you would need to add the Robustness Evaluation as per the instructions in the email :)